apps/shopifypartners/shopify_partners.star

```diff
@@ -0,0 +1,375 @@
+"""
+Applet: Shopify Partners
+Summary: Show Shopify app data
+Description: Show your Shopify app data.
+Author: Rishabh Tayal
+"""
+
+load("animation.star", "animation")
+load("cache.star", "cache")
+load("encoding/base64.star", "base64")
+load("encoding/json.star", "json")
+load("http.star", "http")
+load("render.star", "render")
+load("schema.star", "schema")
+load("time.star", "time")
+
+# CONFIG
+SHOPIFY_COUNTER_API_HOST = "https://www.partnermetrics.io/chart_data?chart_type%5Bcalculation%5D=sum&chart_type%5Bcolumn%5D=revenue&chart_type%5Bdirection_good%5D=up&chart_type%5Bdisplay%5D=currency&chart_type%5Bmetric_type%5D=recurring_revenue&chart_type%5Btitle%5D=Revenue&chart_type%5Btype%5D=recurring_revenue&date="
+
+CACHE_TTL = 30
```

Comment:
Would it be OK to use a (much) larger TTL?

Reply:
Increased to 3600.